content/metadata.yaml

```diff
@@ -6,32 +6,20 @@ keywords:
   - Computational audits
 lang: en-US
 authors:
-  - github: arielah
-    name: Ariel A. Hippen*
+  - name: Ariel A. Hippen*
     orcid: 0000-0001-9336-6543
-    twitter: ariel_hippen
     email: ariel.hippen@gmail.com
     affiliations:
       - Department of Systems Pharmacology and Translational Therapeutics, Perelman School of Medicine, University of Pennsylvania
```

Comment:
My best guess is "Perelman School of Medicine, University of Pennsylvania, 3400 Civic Center Boulevard, Philadelphia, PA 19104"

Reply:
got it, added in https://github.com/greenelab/disparities_opinion/pull/16/commits/7133a790a31730fe67e5c6bc7d3e2b811e2d4b3f